internal/aws/client/client.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"context"
+	"github.com/Falco-Talon/falco-talon/utils"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+)
+
+type AWSClient struct {
+	cfg aws.Config
+
+	mu           sync.Mutex // Protects the fields below
+	lambdaClient *lambda.Client
+	//s3Client     *s3.Client
```

Comment:
I like this too, I would like to prepare an S3 notifier later :wink:

Reply:
We can definitely work on that and leverage the existing aws client here :)